src/main/scala/mesosphere/chaos/example/Main.scala

```diff
@@ -1,23 +1,21 @@
 package mesosphere.chaos.example
 
 import org.rogach.scallop.ScallopConf
-import mesosphere.chaos.http.{ HttpService, HttpConf, HttpModule }
+import mesosphere.chaos.http.{HttpConf, HttpModule}
 import mesosphere.chaos.metrics.MetricsModule
 import mesosphere.chaos.{ App, AppConfiguration }
 
+// TODO: put the whole example somewhere else
 object Main extends App {
-
-  //Declare all Guice Modules
-  def modules() = {
-    Seq(
-      new HttpModule(conf),
-      new MetricsModule,
-      new ExampleRestModule)
-  }
-
   //The fact that this is lazy, allows us to pass it to a Module
   //constructor.
   lazy val conf = new ScallopConf(args) with HttpConf with AppConfiguration
+  conf.afterInit()
 
-  run(classOf[HttpService])
+  lazy val metricRegistry = new MetricsModule(conf).registry
```

Comment:
Proposal: add a chaos module, that binds all chaos modules (metrics, http...), so in main you only create the chaos module + the application (example) module.